### ISSUE TITLE
build_with_gestures build broken for release v2.2.1

### ISSUE THREAD

**Issue by idelsink** [kangax/fabric.js#4806]
## Version
2.2.1

## Steps to reproduce
1. Clone the repo and checkout version `v2.2.1`
```sh
git clone https://github.com/kangax/fabric.js.git && git checkout v2.2.1
```
2. Install and build the tests:
```
npm install
npm run build_with_gestures
```
## Expected Behavior
Passing build.

## Actual Behavior
I get the following error:
```
npm run build_with_gestures

> fabric@2.2.1 build_with_gestures /home/ingmar/workspace/fabric.js
> node build.js modules=ALL exclude=json

Built distribution to dist/fabric.js
/home/ingmar/workspace/fabric.js/build.js:90
      if (err) throw err;
               ^

Error: ENOENT: no such file or directory, open '/home/ingmar/workspace/fabric.js/src/amd/requirejs.js'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! fabric@2.2.1 build_with_gestures: `node build.js modules=ALL exclude=json`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the fabric@2.2.1 build_with_gestures script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ingmar/.npm/_logs/2018-03-12T14_59_18_773Z-debug.log
```

## Additional notes

The latest commit from master (1fa3fdae) builds without any issues.
You can also see that my buildserver, which runs on a cron job, failed the build: https://travis-ci.org/idelsink/fabric.js-autobuild/builds/352272713

**Comment by asturur** [comment]
i ll check this, thanks

**Comment by asturur** [comment]
This will be fixed with the release of 2.2.2
